Mobile_version.css

```diff
@@ -41,6 +41,52 @@ ul {
   text-decoration: none;
 }
 
+.Popup {
+  height: 5130px;
+  width: 375px;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  position: fixed;
+  background-color: #6070ff;
+  z-index: 1;
+  display: none;
+}
+
+.cancel {
+  position: absolute;
+  left: 320px;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+}
+
+.box {
+  display: flex;
+  flex-direction: column;
+  left: 30px;
+  top: 150px;
+  gap: 30px;
+  height: 24px;
+  position: absolute;
+}
+
+.cont,
+.Ab,
+.port {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600px;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+#line {
+  position: absolute;
+  top: 800px;
+  left: 120px;
+}
```

Comment:
**OPTIONAL** - To help improve the UX of your portfolio, it is advisable that your site fits across all mobile screens i.e all screens `<768px`. Please fix this using  CSS and media-queries 🙏 

Your project:
<img width="279" alt="vhvvvh" src="https://user-images.githubusercontent.com/73607512/210355420-1c40cabc-5dac-47c0-b284-74d08e81b6fd.PNG">